packages/chaire-lib-backend/src/utils/osrm/OSRMMode.ts

```diff
@@ -66,14 +67,17 @@ class OSRMMode {
         }
     }
 
-    public async route(params: transitionRouteOptions): Promise<Status.Status<osrm.RouteResults>> {
+    public async route(
+        params: transitionRouteOptions,
+        routingAttributes?: TripRoutingQueryAttributes
+    ): Promise<Status.Status<osrm.RouteResults>> {
```

Comment:
Instead of adding a new parameter, we already have the params object which include the boolean for alternatives. We should probably fill it in there before calling this function.

Reply:
I will add it to the transitionRouteOptions as a new property, but we won't use the one from  osrm as it should be eventually rewritten (see #1115)